src/Symfony/Component/Debug/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.7.1
+-----
+
+* fix support for PHPDBG SAPI in `Debug::enable()`
```

Comment:
This must be reverted. We only add new entries on minor versions, for features.

Reply:
Done.